pkg/kubelet/pod_workers.go

```diff
@@ -105,6 +163,13 @@ func (p *podWorkers) managePodLoop(podUpdates <-chan workUpdate) {
 			}
 			minRuntimeCacheTime = time.Now()
 
+			backoffInterval, lastIncreaseTime = computeBackoffTime(summary, backoffInterval, firstSync, lastIncreaseTime)
+			if backoffInterval > 0 {
```

Comment:
We can probably do the firstSync logic here and not add it to computeBackofftime (we ignore it's output on the firstSync)